### PR TITLE
Style landing page

### PR DESCRIPTION
**Pull request by keiffer01**
### What is a quick description of the change?

Styles the landing page.

### Is this fixing an issue?

none

### Are there more details that are relevant?

Uses the Jumbotron from React Bootstrap and a stock image as the background image (its [license](https://unsplash.com/license) allows it to be used for this case).

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [x] Coding style (indentation, etc)

No tests written, just styling.

### Any additional comments?

Before:
Screenshot 2020-07-28 at 9 10 42 PM

After:
Screenshot 2020-07-30 at 11 53 54 AM

### CODE REVIEW

File: frontend/src/styles/landing.css [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+  transform: translate(-50%, -50%);
+}
+
+#title {
+  font-size: 10vh;
+  margin: 0;
+}
+
+#subtitle {
+  color: var(--SLURP-GRAY);
```
Comment (ananyeet) [inline frontend/src/styles/landing.css:18]: grey is spelled with an e

File: frontend/src/styles/landing.css [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+  font-size: 3vh;
+  opacity: 50%;
+}
+
+#landing-description {
+  color: var(--SLURP-BLUE);
+  font-size: 2vh;
```
Comment (ananyeet) [inline frontend/src/styles/landing.css:25]: Did you test this on multiple window sizes?

File: frontend/src/styles/style.css [hunk 3]
```diff
@@ -8,3 +8,18 @@
...
 }
+
+/* Set dynamic content to use the entirety of the window. */
+html, body, #root, #root>div, #root>div>* {
+  height: 100%
+}
+
+/* Pages with the hero image spans the entire window with a slight dark tint. */
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)),
```
Comment (ananyeet) [inline frontend/src/styles/style.css:19]: Can you write in a `/*comment*/` what this does?

File: frontend/src/styles/landing.css (hunk 1 above)
Comment (keiffer01) [reply frontend/src/styles/landing.css:18]: This is the name from the SLURP color palette.
I am sorry but we are using the american spelling. 😔

File: frontend/src/styles/landing.css (hunk 2 above)
Comment (keiffer01) [reply frontend/src/styles/landing.css:25]: Yeah I played around with multiple window sizes and it looks fine.

File: frontend/src/styles/style.css (hunk 3 above)
Comment (keiffer01) [reply frontend/src/styles/style.css:20]: Added. 👍

**Comment by keiffer01** [comment]
> Why is the sign in button so small and off to the side?

Okay so since you and Hiram think it looks off I modified it slightly, the image in the PR description has been updated. Does that look better?

**Pull request merged**